forge-ai/src/main/java/forge/ai/ComputerUtilMana.java

```diff
@@ -1521,7 +1523,17 @@ public boolean apply(final Card c) {
                 final Cost cost = m.getPayCosts();
 
                 if (cost != null) {
-                    needsLimitedResources |= !cost.isReusuableResource();
+                    if (!cost.isReusuableResource()) {
```

Comment:
`ComputerUtilCard.isSacrificeSelfCost`

Reply:
That function doesn't really help me. I need to check if its a sacrifice Cost AND not a self sacrifice cost.